src/com/griddynamics/main/Ticket.java

```diff
@@ -0,0 +1,41 @@
+package com.griddynamics.main;
+
+import java.util.Scanner;
+
+public class Ticket {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.println("Please enter: ");
+        String ticket = in.nextLine();
+        isMyTicketLucky(ticket);
+        in.close();
+    }
+
+    public static boolean isMyTicketLucky(String ticketNumber) throws IllegalArgumentException {
```

Comment:
@kolegova-mary as far as I remember IllegalArgumentException is an unchecked exception so it does not make sense to use throws here.